src/main/java/io/nats/client/impl/MessageQueue.java

```diff
@@ -33,11 +33,13 @@ class MessageQueue {
     protected final AtomicLong length;
     protected final AtomicLong sizeInBytes;
     protected final AtomicInteger running;
-    protected final boolean singleThreadedReader;
+    protected final boolean singleReaderMode;

```

Comment:
made the variable name more consistent with it's usage and to match it's getter.